tools/include/eosio/gen.hpp

```diff
@@ -541,6 +541,23 @@ struct generation_utils {
          }
          return replace_in_name(ret);
       }
+      else if ( is_template_specialization( type, {"array"} )) {
+         std::string orig = type.getAsString();
+         std:: string ret = "";
+         ret += get_template_argument_as_string( type, 0 );
+         ret = replace_in_name(ret);
+         ret += '[';
+         auto pos1 = orig.find_last_of(',');
+         auto pos2 = orig.find_last_of('>');
+         std::string digits = orig.substr(pos1 + 1, pos2 - pos1 - 1);
+         for(auto & c : digits) {
```

Comment:
After getting substring `digits`, could it just strip (surround) spaces and then append to `ret`? If `digits` contains chars other than digits, earlier phase will pop errors.

Reply:
Good idea,  std::remove is used instead of for loop.